tests/test_strength_analysis_integration_real.py

```
@@ -0,0 +1,261 @@
+"""
+強み分析Celeryタスクの実際の統合テスト
+
+Testing Trophy戦略に従い、実際のCeleryコンポーネントを使用した統合テスト。
+インメモリブローカーを使用し、current_taskのモックを最小限に留める。
+"""
+import pytest
+import json
+from celery.contrib.testing.worker import start_worker
+from celery.contrib.testing.app import TestApp
```

Ruff (0.12.2)

`celery.contrib.testing.app.TestApp` imported but unused

Remove unused import: `celery.contrib.testing.app.TestApp`

(F401)